### PR TITLE
Swagger added to the project

### PR DESCRIPTION
**Pull request by ZedZipDev**
I have added the Swagger package and Swagger filter (SWFilter.cs) which helps to add tenant id to the HTTP header

### CODE REVIEW

File: CSharp/AspNetCoreMultitenancy/Startup.cs [hunk 1]
```diff
@@ -57,6 +63,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+            app.UseSwagger();
```
Comment (bytefish) [inline CSharp/AspNetCoreMultitenancy/Startup.cs:66]: Could we move this into the `IsDevelopment()` part some lines above, like:

```csharp
 if (env.IsDevelopment())
{
    // Swagger goes here
}
//...
```

File: CSharp/AspNetCoreMultitenancy/Startup.cs [hunk 2]
```diff
@@ -26,13 +27,18 @@ public void ConfigureServices(IServiceCollection services)
             // Register Scoped DbContexts:
             services
                 // Register the Tenant Database:
-                .AddDbContext<TenantDbContext>(options => options.UseNpgsql("Host=localhost;Port=5432;Database=sampledb;Pooling=false;User Id=app_user;Password=app_user;"))
+                .AddDbContext<TenantDbContext>(options => options.UseNpgsql("Host=d-bkp2;Port=5432;Database=sampledb;Pooling=false;User Id=app_user;Password=app_user;"))
```
[the remark this answers is not in the document]
Comment (ZedZipDev) [reply CSharp/AspNetCoreMultitenancy/Startup.cs]: Done, and yes, I will move it to appsettings.json

**Comment by ZedZipDev** [comment]
Hey Philipp, I have made all what was you wrote and + something else ;-)

**Comment by bytefish** [comment]
I have made some minor changes, such as using the "ConnectionStrings" section in the appsettings.json and renaming the Operation Filter. Thanks for all your work! 👍

**Pull request merged**